src/client/Microsoft.Identity.Client/Cache/Items/MsalAccessTokenCacheItem.cs

```diff
@@ -31,7 +36,8 @@ internal MsalAccessTokenCacheItem(
             string homeAccountId,
             string keyId = null,
             string oboCacheKey = null,
-            IEnumerable<string> persistedCacheParameters = null)
+            IEnumerable<string> persistedCacheParameters = null,

```

Comment:
Do the cacheKeyComponents behave like the persistedCacheParameters? Should they?

Reply:
Yes, both are stored in the payload and should be treated the same.